File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/AfpHistorikk.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.AfpOrdningTypeCti;

import java.io.Serializable;
import java.util.Date;

public class AfpHistorikk implements Serializable {

    /**
     * Fremtidig pensjonspoeng
     */
    private double afpFpp;

    private Date virkFom;

    private Date virkTom;

    private int afpPensjonsgrad;

    private AfpOrdningTypeCti afpOrdning;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/AfpHistorikk.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.enum.AFPtypeEnum
import java.io.Serializable
import java.util.*
import java.time.LocalDate

class AfpHistorikk : Serializable {
    /**
     * Fremtidig pensjonspoeng
     */
    var afpFpp = 0.0
    var virkFomLd: LocalDate? = null
    var virkTomLd: LocalDate? = null
    var afpPensjonsgrad = 0
    var afpOrdningEnum: AFPtypeEnum? = null

}